src/main/docker/docker-entrypoint.d/89-check-validity.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+currentDate=`date +%s`
+if [ -z "${BUILD_DATE}" ]
+then
+    1>&2 echo "************"
+    1>&2 echo "This is an old image: BUILD_DATE is not set"
+    1>&2 echo "*************"
```

Comment:
Number of stars on the top and the bottom does not match.
Is it on purpose to annoy people slightly more that they are using an old image? :smiley: